### PR TITLE
Fix seo issues

### LINKED ISSUE #7: SEO issues from Lighthouse Audit
Screenshot 2020-09-10 at 16 26 23

- [ ] Document does not have a meta description
- [ ] Links do not have descriptive text
- [ ] Tap targets are not sized appropriately

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Add meta description
- Change link name from "Learn more" to "Kubeflow-lite". Lighthouse failed as "learn more" is too generic. [Read more here](https://web.dev/link-text/?utm_source=lighthouse&utm_medium=devtools)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8046
- Run the lighthouse audit and check SEO audit is 100. 

## Issue / Card

Fixes #7

### CODE REVIEW

**Comment by barrymcgee** [comment]
@bethcollins92 Looks good but you have a conflict to resolve before this can land.

**Comment by bethcollins92** [comment]
@barrymcgee Sorted that conflict, thanks. Let me know if it's ok to merge.

File: templates/index.html [hunk 1]
```diff
@@ -382,7 +382,7 @@ <h2>Lightweight workstation deployments</h2>
       <p>With the expansion of Kubeflow as a full ML toolkit, running it on desktops with up to 16Gb of RAM has become a painful experience.</p>
       <p>Kubeflow-lite is a lightweight version of Kubeflow with the ML services you typically need on your desktop, on a smooth experience.</p>
       <p>
-        <a href="https://jaas.ai/u/kubeflow-charmers/kubeflow-lite" class="p-link--external">Learn more</a>
+        <a href="https://jaas.ai/u/kubeflow-charmers/kubeflow-lite" class="p-link--external">Kubeflow-lite</a>
```
Comment (anthonydillon) [inline templates/index.html:385]:
```suggestion
        <a href="https://ubuntu.com/kubeflow/install" class="p-button--positive p-link--external">Install Kubeflow lite</a>
        <a href="https://jaas.ai/u/kubeflow-charmers/kubeflow-lite" class="p-link--external">Learn more about Kubeflow-lite</a>
```

Comment (anthonydillon) [reply templates/index.html:385]: Updated the copy doc to match this change. https://docs.google.com/document/d/1Wvhl0yYV_w0BJsyZbQFKg_QV6RjMLbmQU3x0rsD2F4Q/edit#heading=h.mvjb0cu9g00d

**Review by anthonydillon — CHANGES_REQUESTED** [review]
One suggestion in the code.

File: templates/index.html (hunk 1 above)
Comment (bethcollins92) [reply templates/index.html:385]: New PR here [https://github.com/canonical-web-and-design/charmed-kubeflow.io/pull/17](https://github.com/canonical-web-and-design/charmed-kubeflow.io/pull/17)

**Pull request closed without merging**